packages/cli/templates/actions/empty-browser-action/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import { Analytics, Context } from '@segment/analytics-next'
+import * as jsdom from 'jsdom'
+import Destination from '../index'
```

Comment:
This will need to be: 
```suggestion
import { destination } from '../index'
```